### PR TITLE
Example pages for LocalPreview, DeviceSettings, TeamsTheme, and Incoming Call Toast/Modal

### PR DESCRIPTION
**Pull request by alkwa-msft**

### CODE REVIEW

**Comment by dmceachernmsft** [comment]
image
Something is wrong with the icons on the local preview, otherwise looks good!

**Review by dmceachernmsft — APPROVED** [review]

File: packages/storybook8/stories/Examples/Themes/__snapshots__/TeamsTheme.stories.storyshot
Comment (prabhjot-msft) [inline packages/storybook8/stories/Examples/Themes/__snapshots__/TeamsTheme.stories.storyshot:1]: We don't have storyshot in v8, these should be removed

File: packages/storybook8/stories/Examples/LocalPreview/__snapshots__/LocalPreview.stories.storyshot
Comment (prabhjot-msft) [inline packages/storybook8/stories/Examples/LocalPreview/__snapshots__/LocalPreview.stories.storyshot:1]: We don't have storyshot in v8, these should be removed

File: packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallToast/__snapshots__/IncomingCallToast.stories.storyshot
Comment (prabhjot-msft) [inline packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallToast/__snapshots__/IncomingCallToast.stories.storyshot:1]: We don't have storyshot in v8, these should be removed

File: packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallModal/__snapshots__/IncomingCallModal.stories.storyshot
Comment (prabhjot-msft) [inline packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallModal/__snapshots__/IncomingCallModal.stories.storyshot:1]: We don't have storyshot in v8, these should be removed

**Review by prabhjot-msft — COMMENTED** [review]
For local review, video is not showing and sometimes buttons are disabled
Screenshot 2024-08-02 at 12 00 59 PM

For incoming call modal, video is not showing up:
Screenshot 2024-08-02 at 12 04 46 PM
Screenshot 2024-08-02 at 12 05 05 PM

For incoming call toast, styling issue
Screenshot 2024-08-02 at 12 05 54 PM
Screenshot 2024-08-02 at 12 06 00 PM

File: packages/storybook8/stories/Examples/Themes/__snapshots__/TeamsTheme.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/Themes/__snapshots__/TeamsTheme.stories.storyshot:1]: will remove

File: packages/storybook8/stories/Examples/LocalPreview/__snapshots__/LocalPreview.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/LocalPreview/__snapshots__/LocalPreview.stories.storyshot:1]: will remove

File: packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallModal/__snapshots__/IncomingCallModal.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallModal/__snapshots__/IncomingCallModal.stories.storyshot:1]: will remove

File: packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallToast/__snapshots__/IncomingCallToast.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallToast/__snapshots__/IncomingCallToast.stories.storyshot:1]: fixed

File: packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallModal/__snapshots__/IncomingCallModal.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/IncomingCallAlerts/IncomingCallModal/__snapshots__/IncomingCallModal.stories.storyshot:1]: fixed

File: packages/storybook8/stories/Examples/LocalPreview/__snapshots__/LocalPreview.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/LocalPreview/__snapshots__/LocalPreview.stories.storyshot:1]: fixed

File: packages/storybook8/stories/Examples/Themes/__snapshots__/TeamsTheme.stories.storyshot
Comment (alkwa-msft) [reply packages/storybook8/stories/Examples/Themes/__snapshots__/TeamsTheme.stories.storyshot:1]: fixed

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**